test_runner.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from click.testing import CliRunner
+from runner import run, Command
+
+
+def test_should_fail_to_run_cmd_multiple_time():
+    command = Command(cmd='ping', attempts=3)
+    command.execute(repeat_times=3)
+    assert command.attempts == 0
```

Comment:
read this line out loud. doesn't really rings correctly right?
what does '0 attempts' mean?

it would have been much better if you had inside the command both the number of wanted retries, and the number of actual attempts. then you can assert they are equal.